README.md

```diff
@@ -1,83 +1,87 @@
 # nebula-python
 
-Nebula client API in Python
+This repository provides Nebula client API in Python.
 
-## the directory structure
+## The directory structure
 
 ```text
 |--nebula-python
     |
     |-- nebula                        // client code
-    |   |-- Client.py                 // the interfaces to operate nebula 
-    |   |-- ConnectionPool.py         // manger all connection, user can change the connection nums and timeout
+    |   |-- Client.py                 // interfaces controlling nebula
+    |   |-- ConnectionPool.py         // the connection pool that manages all the connections, users can specify the connection numbers and timeout when creating it
     |
-    |-- common                        // the common types
+    |-- common                        // the common data types
     |
-    |-- graph                         // the types with graphd, and the client with graphd
+    |-- graph                         // date types and client interfaces interact with graphd
     |
-    |-- meta                          // the types with metad, and the client with metad
+    |-- meta                          // date types and client interfaces interact with metad
     |
-    |-- raftex                        // the types with raftex, and the client with raftex
+    |-- raftex                        // date types and client interfaces interact with raftex
     |
-    |-- storage                       // the types with stroged, and the client with stroged
+    |-- storage                       // date types and client interfaces interact with storaged
     |
-    |__ thrift                        // the thrift code, about the socket
-    |   
-    |-- examples               
-    |   |__ ClientExample.py          // the example
+    |__ thrift                        // the socket implementation code
+    |
+    |-- examples
+    |   |__ ClientExample.py          // the example code
     |
     |-- tests
-    |   |__ test_client.py            // the test file
+    |   |__ test_client.py            // the test code
     |
-    |-- setup.py                      // use to install or package
+    |-- setup.py                      // used to install or package
```

Comment:
use

Reply:
`setup.py` is used to install or package